### DISCUSSION TITLE
Problème avec ATC en Français pour msfs

### DISCUSSION THREAD

**Discussion by LEGALLAIS40** [Nash115/frenchATC-for-MSFS2020#22] [Help]
Lorsque je commence la contact, ATC me répond "Fox Echo Lima bonjour"
Lorsque je transmets le deuxième message avec la phraséologie, je n'ai aucune réponse ...

**Comment by Nash115** [comment]
Bonjour, pour mieux comprendre votre problème, il me faudrait plus d'informations. La reconnaissance vocale a l'air de fonctionner. Que prononcez vous après le premier contact ?

**Comment by LEGALLAIS40** [comment]
Bonjour,
Lorsque je me présente  "Fox Echo Lima, bonjour"La réponse : "Fox Echo Lima, transmettez votre message"Je transmets : "Fox Echo Lima, un Cessna 145, 2 personnes à bord, pour un départ dans l'axe"Réponse ; "Fox Echo Lima, je n'ai pas compris votre message"
Merci pour votre réactivité.
Cordialement Eric LEGALLAIS
    Le jeudi 11 mai 2023 à 19:37:07 UTC+2, nash115 ***@***.***> a écrit :  
 
 


Bonjour, pour mieux comprendre votre problème, il me faudrait plus d'informations. La reconnaissance vocale a l'air de fonctionner. Que prononcez vous après le premier contact ?

**Reply by Nash115** [reply]
Très bien.
La fréquence sur laquelle vous transmettez est-elle bien une fréquence "sol" ? Car par exemple, si la fréquence est une fréquence "tour" uniquement, effectivement le programme vous renverra cette réponse.
Si c'est le cas, vous devrez paramétrer correctement les fréquences dans le fichier "airports-locations.json". Votre aéroport devrait contenir au moins une fréquence "grd" (ou "all"), même si elle est identique a celle de la tour.
